### PR TITLE
Added a 'To Read' tag

### PR DESCRIPTION
**Pull request by prabhav-pandya**
**Pull request is linked to**    
Issue #14

**Changes proposed in this pull request**   
Replaced 'On Hiatus' with 'To Read' and sort by 'On Read'

### CODE REVIEW

File: src/lib/details_page/components/chapter_progress.dart [hunk 1]
```diff
@@ -290,10 +290,10 @@ class NovelStatus extends StatelessWidget {
           ),
           onTap: () {
             if (novelStatus == 'Complete') {
-              updateStatus('On Hiatus');
+              updateStatus('To Read');
```
Comment (OverPoweredDev) [inline src/lib/details_page/components/chapter_progress.dart:293]: No not this, we wanna keep On Hiatus as a Status option. Just remove it from the sortOptions

File: src/lib/homepage/body.dart [hunk 2]
```diff
@@ -145,14 +145,14 @@ class _HomePageState extends State<HomePage> {
           return sortValue;
         });
         break;
-      case 'On Hiatus':
+      case 'To Read':
```
Comment (OverPoweredDev) [inline src/lib/homepage/body.dart:148]: This part's right

File: src/lib/details_page/components/chapter_progress.dart (hunk 1 above)
Comment (prabhav-pandya) [reply src/lib/details_page/components/chapter_progress.dart:293]: alright

**Review by OverPoweredDev — CHANGES_REQUESTED** [review]
Just check a few things. Add `To Read` as a circular option after `Complete` and before `On Hiatus` and remove it _only_ from the `sortOptions`, not `NovelStatus()`

File: src/lib/details_page/components/chapter_progress.dart (hunk 1 above)
Comment (prabhav-pandya) [reply src/lib/details_page/components/chapter_progress.dart]: Okay, it's fixed now.

**Review by OverPoweredDev — APPROVED** [review]
Yep, Everything looks great!

**Pull request merged**